sass/_emoji.scss

```diff
@@ -94,7 +97,11 @@
                         }
                         a {
                             padding: 3px;
+                            display: inline-block;
                             font-size: var(--font-size-huge);
+                            height: 30px;
+                            width: 30px;
```

Comment:
In changing the width from 32px to 30px, the emoji is no longer centered within the `.insert-emoji` element which gets a background color upon hover. Please fix.

![Screenshot from 2020-05-27 07-27-17](https://user-images.githubusercontent.com/683911/82981220-c20d3d80-9feb-11ea-91b4-8e92dc805708.png)

Reply:
Are those image-based emojis or is that how the regular ones render in Linux?

Reply:
These are system (regular) emojis on Linux.

Reply:
Ok check now, I tested with system emojis on Mac and also image emojis by updating the settings. Lmk what it looks like on Linux.

Reply:
Looks worse to me now, on both Linux and Mac.

Can you please post a screenshot of what it looks like on your machine?

Reply:
I was looking at the fullscreen view, but I was missing the fact that the overlay version has different emoji sizes. So I just fixed that. Here is what overlay and fullscreen look like now

<img width="674" alt="Screen Shot 2020-06-01 at 12 12 53 PM" src="https://user-images.githubusercontent.com/7013479/83435092-3b989780-a401-11ea-837f-7dd5cb5e8508.png">
<img width="730" alt="Screen Shot 2020-06-01 at 12 13 48 PM" src="https://user-images.githubusercontent.com/7013479/83435156-5cf98380-a401-11ea-8ea0-7082f5ca404f.png">